using-timescaledb/data-retention.md

```diff
@@ -147,72 +69,38 @@ SELECT remove_retention_policy('conditions');
 
 The scheduler framework also allows one to view scheduled jobs:
 ```sql
-SELECT * FROM timescaledb_information.drop_chunks_policies;
+SELECT * FROM timescaledb_information.policy_stats;
```

Comment:
This will probably be renamed to job_stats but final decission is still pending

Reply:
Update to `job_stats`